src/megatron/bridge/recipes/deepseek/deepseek_v3.py

```diff
@@ -184,6 +178,12 @@ def pretrain_config(
     precision_config: Optional[Union[MixedPrecisionConfig, str]] = None,
     comm_overlap_config: Optional[CommOverlapConfig] = None,
     enable_deepep: bool = False,
+    # Recomputation
+    recompute_granularity: str = "selective",
+    recompute_modules: Optional[List[str]] = None,
+    recompute_method: Optional[str] = None,
+    recompute_num_layers: Optional[int] = None,
+    apply_rope_fusion: bool = False,
```

Comment:
"false" currently has better perf because of output spamming of experimental feature. Once that mcore MR is merged, i will reevaluate